docs/docs/overview/quickstart/get-started-with-nextjs.md

```diff
@@ -201,44 +201,44 @@ You can create a topic using two entities - `key` and `name`. Keys are unique id
 
 ```jsx
 import { Novu } from '@novu/node';
-import CircularJSON from 'circular-json';
 
 export default async function createTopic(req, res) {
     try {
-        const novu = new Novu(<YOUR_NOVU_API_KEY>);
+        const novu = new Novu(process.env.NOVU_API_KEY);
```

Comment:
Quick question. Does the quickstart codebase actually use an env variable? If it is, good. If it does not, then this needs to be reverted back to `YOUR_NOVU_API_KEY`